src/Microsoft.ComponentDetection.Detectors/cocoapods/PodComponentDetector.cs

```diff
@@ -389,6 +390,7 @@ private static (Pod pod, string key, DetectedComponent detectedComponent)[] Read
                     && checkoutOptions.TryGetValue(":commit", out string commitOption))
                 {
                     // Create the Git component
+                    gitOption = NormalizePodfileGitUri(gitOption);
                     typedComponent = new GitComponent(new Uri(gitOption), commitOption);
```

Comment:
Since this is introducing new values for same components, I think we should bump the detector version.